lib/IRGen/StructLayout.cpp

```diff
@@ -87,7 +87,8 @@ StructLayout::StructLayout(IRGenModule &IGM, std::optional<CanType> type,
   if (rawLayout && type) {
     auto sd = cast<StructDecl>(decl);
     IsKnownTriviallyDestroyable = triviallyDestroyable;
-    IsKnownBitwiseTakable = bitwiseTakable;
+    // Raw layout types are never bitwise-borrowable.
+    IsKnownBitwiseTakable = bitwiseTakable & IsBitwiseTakableOnly;
```

Comment:
Would this be more logically written
` IsKnownBitwiseTakable = bitwiseTakable >= IsBitwiseTakableOnly`?

Reply:
I was trying to keep the patch size down, but now that `IsBitwiseTakable_t` is a tristate, I want to rip the `Is` off of these `Is*BitwiseTakable` names. The code here is masking out the `IsBitwiseTakableAndBorrowable` possibility since a raw layout type can be at most bitwise-takable without -borrowable.